.buildkite/premerge.steps.yaml

```diff
@@ -73,16 +73,28 @@ steps:
     artifact_paths:
       - logs/**/*
 
+  - label: ":darwin: ~ test"
+    id: "test-darwin"
+    command: bash -c ci/test.sh
+    <<: *macos
+    artifact_paths:
+      - logs/**/*
+
   - label: ":debian: ~ annotate test results :pencil2:"
     depends_on:
       - step: "test-windows"
         allow_failure: true
+      - step: "test-darwin"
+        allow_failure: true
     plugins:
       - improbable/test-summary#d7289cac8297018fd1b452dcf828979307b3ebc6:
           inputs:
             - label: ":octagonal_sign: :windows: Test failures "
               artifact_path: "logs\\nunit\\*.xml"
               type: nunit
+            - label: ":octagonal_sign: :darwin: Test failures "
+              artifact_path: "logs/nunit/*.xml"
+              type: nunit
```

Comment:
Its funny that this oddity in BuildKite actually helps us out here

Reply:
I'm not 100% sure this actually works as intended.

Reply:
Force a test failure (in different ways for each platform) and see what happens?

Reply:
Just going to enable the test first. will test do another pass for the annotation afterwards